Tests/SpriteKitTests/SKNodeExtensionTests.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  SpriteKitTests.swift
+//  SwifterSwift
+//
+//  Created by Olivia Brown on 5/28/18.
+//  Copyright © 2018 SwifterSwift
+//
+
+import XCTest
+@testable import SwifterSwift
+
+#if canImport(SpriteKit)
+import SpriteKit
+
+@testable import SwifterSwift
```

Comment:
You can remove this.

Reply:
Good spot, I've got no idea why it would be there in the original file. 
🧹 💨